### PR TITLE
Add slack channel for Karpenter

### PR DESCRIPTION
**Pull request by ellistarn**
**Which issue(s) this PR fixes**:
See relevant discussion: https://kubernetes.slack.com/archives/C4M06S5HS/p1641263233066400?thread_ts=1641229254.066200&cid=C4M06S5HS

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -157,6 +157,7 @@ channels:
   - name: carvel
     id: CH8KCCKA5
   - name: kalm
+  - name: karpenter
```
Comment (coderanger) [inline communication/slack-config/channels.yaml:160]: We try to keep this in alpha sort order, can you move this down one line?

**Review by DylanGraham — APPROVED** [review]
/approve

**Comment by coderanger** [comment]
/approve

Need to get @DylanGraham into the OWNERS file.

**Pull request merged**